### PR TITLE
Improve the README docu for extending Kafka connect with additional p…

### PR DESCRIPTION
**Pull request by scholzj**
…lugins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,8 +193,11 @@ cluster as service `kafka-connect` on port `8083`.
 
 ### Using Kafka Connect with additional plugins
 
-Our Kafka Connect images contain by default only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-If you want to use other connectors, you can:
+Our Kafka Connect Docker images contain by default only the `FileStreamSinkConnector` and 
+`FileStreamSourceConnector` connectors which are part of the Apache Kafka project.
+
+Kafka Connect will automatically load all plugins / connectors which are present in the `/opt/kafka/plugins`
```
Comment (ppatierno) [inline README.md]: I'm not using spaces before and after the "/" character. But it's just a choice. Let's do it in the same way, with or without spaces.

File: README.md [hunk 2]
```diff
@@ -203,9 +206,66 @@ If you want to use other connectors, you can:
 
 #### Mount a volume containing the plugins
 
-1. Prepare a PersistentVolume which contains a directory with your plugin(s) and their dependencies and ensure 
-   these files are world-readable (`chmod -R a+r /path/to/your/directory`).
-2. Mount the volume into your Pod at the path `/opt/kafka/plugins/`
+You can create Kubernetes / OpenShift volume which contains additional plugins and modify the Kafka Connect
```
Comment (ppatierno) [inline README.md]: Ditto as above.

File: README.md [hunk 3]
```diff
@@ -203,9 +206,66 @@ If you want to use other connectors, you can:
...
+Alternatively you can distribute your plugins to all your cluster nodes into the same path, make them 
+world-readable (`chmod -R a+r /path/to/your/directory`) and use the hostPath volume to mount them into 
+your Kafka Connect deployment. To do so use the following volume specification with the example above.
+
+```yaml
+volumes:
+  - name: pluginsvol
+    hostPath:
+      path: /path/to/my/plugins
+      type: Directory
```
Comment (ppatierno) [inline README.md]: Maybe it's better to use the `hostpath` as the main example for doing that and leaving the GlusterFS as an example of alternatives ?

File: README.md [hunk 4]
```diff
@@ -203,9 +206,66 @@ If you want to use other connectors, you can:
...
+You can create Kubernetes / OpenShift volume which contains additional plugins and modify the Kafka Connect 
+deployment to use this volume. Since distributed Kafka Connect cluster can run on multiple nodes you need to 
+make sure that the volume can be mounted as read only into multiple pods at the same time. Which volume types 
+can be mounted read only on several pods can be found in [Kubernetes documentation][k8srwomany]. Once you have 
+such volume, you can edit the deployment YAML file according to the example bellow:
+
+1. Open the [OpenShift template](kafka-connect/resources/openshift-template.yaml) 
+or [Kubernetes deployment](kafka-connect/resources/kafka-connect.yaml)
+2. add the `volumeMounts` and `volumes` sections in the same way as in the example below
+3. Redeploy Kafka Connect for the changes to take effect
```
Comment (ppatierno) [inline README.md]: Does the redeploy sound like "delete the previous deployment and then deploy the new one". Is it what are you meaning ?

File: README.md (hunk 1 above)
Comment (scholzj) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (scholzj) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (scholzj) [reply README.md]: Fixed.

File: README.md (hunk 3 above)
Comment (scholzj) [reply README.md]: Fixed.

File: README.md [hunk 5]
```diff
@@ -203,9 +206,69 @@ If you want to use other connectors, you can:
...
+You can distribute your plugins to all your cluster nodes into the same path, make them 
+world-readable (`chmod -R a+r /path/to/your/directory`) and use the hostPath volume to mount them into 
+your Kafka Connect deployment. To use the volume, you have to edit the deployment YAML files:
+
+1. Open the [OpenShift template](kafka-connect/resources/openshift-template.yaml) 
+or [Kubernetes deployment](kafka-connect/resources/kafka-connect.yaml)
+2. add the `volumeMounts` and `volumes` sections in the same way as in the example below
+3. Redeploy Kafka Connect for the changes to take effect (If you have Kafka Connect already deployed, you have to apply 
+1. the changes to the deployment and afterwards make sure all pods are restarted. If you haven't yet deployed Kakfa Connect, 
+1. just follow the guide above and use your modified YAML files.)
```
Comment (ppatierno) [inline README.md]: the latest "1." create two more bullet points in the list, while the last one should be 3.

Comment (scholzj) [reply README.md]: Fixed

**Pull request merged**